src/unix/ibus/key_event_handler.cc

```diff
@@ -38,6 +38,10 @@ namespace mozc {
 namespace ibus {
 
 namespace {
+
+// Mod1 is Alt, which is used by Mozc.
+const uint kExtraModMask = IBUS_MOD2_MASK | IBUS_MOD3_MASK | IBUS_MOD4_MASK | IBUS_MOD5_MASK;
```

Comment:
Let's use constexper instead of const, and move this into the function.
Also please wrap the line within 80 chars.

```
  // Mod1 is Alt, which is used by Mozc.
  constexpr uint kExtraModMask =
      IBUS_MOD2_MASK | IBUS_MOD3_MASK | IBUS_MOD4_MASK | IBUS_MOD5_MASK;
  if (modifiers & kExtraModMask) {
    return false;
  }
```